sphinxcontrib/jquery/__init__.py

```diff
@@ -7,17 +7,24 @@
 version_info = (2, 0, 0)
 
 _ROOT_DIR = path.abspath(path.dirname(__file__))
+_FILES = (
+    (
+        'jquery.js',
+        'sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK',
+    ),
+    (
+        '_sphinx_javascript_frameworks_compat.js',
+        'sha384-lSZeSIVKp9myfKbDQ3GkN/KHjUc+mzg17VKDN4Y2kUeBSJioB9QSM639vM9fuY//',
+    ),
+)
```

Comment:
This is such a nice pattern, I really like that it's hard-coded and the test verifies that it was correct afterwards :+1: